src/Utilities/version.f90

```diff
@@ -16,8 +16,8 @@ module VersionModule
   ! -- modflow 6 version
   integer(I4B), parameter :: IDEVELOPMODE = 1
   character(len=*), parameter :: VERSIONNUMBER = '6.4.0'
```

Comment:
This was updated to 6.4.1 on develop.  I think we also need to do that here.

Reply:
Tentative solution discussed via DM and reproduced below for posterity:

The release workflow auto-updates version files to the version specified in the name of the branch triggering the workflow. It may not be known at release time whether the next release will increment patch or minor number, so the version number can remain at the last released until auto-updated in a new release. This has the consequence that developers or anybody working from the head of `develop` in between releases will see the last official release's version number. This departs from the previous convention, which was to bump the version number on `develop` immediately after release.